### PR TITLE
Added missing role(ESDTTransferRole) on fungible token

### PR DESCRIPTION
**Pull request by popenta**
Added missing role according to [this PR](https://github.com/multiversx/mx-sdk-specs/pull/65).

### CODE REVIEW

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts [hunk 1]
```diff
@@ -133,6 +133,47 @@ describe("test token management transactions factory", () => {
         assert.deepEqual(transaction.value, config.issueCost);
     });
 
+    it("should create 'Transaction' for setting spcial role on fungible token", () => {
```
Comment (CiprianDraghici) [inline src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:136]: small typo => 'special role'

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts [hunk 2]
```diff
@@ -133,6 +133,47 @@ describe("test token management transactions factory", () => {
...
+            Buffer.from(
+                "setSpecialRole@4652414e4b2d313163653365@1e8a8b6b49de5b7be10aaa158a5a6a4abb4b56cc08f524bb5e6cd5f211ad3e13@45534454526f6c654c6f63616c4d696e74",
+            ),
+        );
+        assert.equal(transaction.sender, frank.address.toString());
+        assert.equal(transaction.receiver, ESDT_CONTRACT_ADDRESS);
+        assert.equal(transaction.value, 0n);
+    });
+
+    it("should create 'Transaction' for setting all spcial roles on fungible token", () => {
```
Comment (CiprianDraghici) [inline src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:156]: small typo => 'special role'

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts [hunk 3]
```diff
@@ -133,6 +133,47 @@ describe("test token management transactions factory", () => {
...
+            Buffer.from(
+                "setSpecialRole@4652414e4b2d313163653365@1e8a8b6b49de5b7be10aaa158a5a6a4abb4b56cc08f524bb5e6cd5f211ad3e13@45534454526f6c654c6f63616c4d696e74@45534454526f6c654c6f63616c4275726e@455344545472616e73666572526f6c65",
+            ),
+        );
+        assert.equal(transaction.sender, frank.address.toString());
+        assert.equal(transaction.receiver, ESDT_CONTRACT_ADDRESS);
+        assert.equal(transaction.value, 0n);
+    });
+
     it("should create 'Transaction' for setting spcial role on non-fungible token", () => {
```
Comment (CiprianDraghici) [inline src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:177]: small typo => 'special role'

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts (hunk 1 above)
Comment (popenta) [reply src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:136]: fixed

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts (hunk 2 above)
Comment (popenta) [reply src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:156]: fixed

File: src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts (hunk 3 above)
Comment (popenta) [reply src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts:177]: fixed

**Review by andreibancioiu — APPROVED** [review]

**Review by CiprianDraghici — APPROVED** [review]

**Pull request merged**